### ISSUE TITLE
Add swagger generation with --watch mode / live reload

### ISSUE THREAD

**Issue by loucass003** [samchon/nestia#703]
Please add a way to support live reloading with nestia

For instance in our project we use the swagger.json file 

```ts
import swaggerDocument from '../swagger.json';

SwaggerModule.setup('/api', app, swaggerDocument as OpenAPIObject, {
    swaggerOptions: {
      defaultModelRendering: 'model',
      tryItOutEnabled: true,
      syntaxHighlight: {
        activate: true,
      },
    },
});
 ```
 
Then we run the ``nestia swagger`` command to regenerate the schema, but it would be nice to have nestia to watch for changes and re generate the schema when a controller change.

This would be a really nice 'ease of use' feature.

I guess we could try to make a webpack plugin that run the command when a controller change but i think it would be better if it was handled by nestia itself.

Thanks again for this great tool <3

**Comment by samchon** [comment]
Good idea, but do not know how to, and not enough ime to study right now.

Hope someone to contribute about this.

**Comment by samchon** [comment]
Fixed by #1502 and merged into the `next` branch.

`nestia swagger --watch` now runs the normal Swagger generator once, then regenerates `swagger.json` when controller, config, or project inputs change. The same option is also available through `npx @nestia/sdk swagger --watch`, and the docs now cover the live reload workflow.

This will be available on the v12 line once TypeScript-Go is officially released.
